packages/@uppy/dashboard/src/components/AddFiles.tsx

```diff
@@ -40,31 +56,35 @@ class AddFiles extends Component {
     // ___Why not use value="" on <input/> instead?
     //    Because if we use that method of clearing the input,
     //    Chrome will not trigger change if we drop the same file twice (Issue #768).
-    event.target.value = null // eslint-disable-line no-param-reassign
+    ;(event.target as HTMLInputElement).value = '' // eslint-disable-line no-param-reassign
   }
 
-  private renderHiddenInput = (isFolder: $TSFixMe, refCallback: $TSFixMe) => {
+  private renderHiddenInput = (
+    isFolder: boolean,
+    refCallback: (ref: HTMLInputElement | null) => void,
+  ) => {
     return (
       <input
         className="uppy-Dashboard-input"
         hidden
         aria-hidden="true"
         tabIndex={-1}
+        // @ts-expect-error default types don't yet know about the `webkitdirectory` property
         webkitdirectory={isFolder}
         type="file"
         name="files[]"
         multiple={this.props.maxNumberOfFiles !== 1}
         onChange={this.onFileInputChange}
-        accept={this.props.allowedFileTypes}
+        accept={this.props.allowedFileTypes?.join(', ')}
```

Comment:
```suggestion
        accept={this.props.allowedFileTypes?.join(',')}
```

Reply:
Why? I went for `', '` because that's easier to read in html.